### DISCUSSION TITLE
Change name the board

### DISCUSSION THREAD

**Discussion by pmjcontroller1** [shanteacontrols/OpenDeck#215] [Q&A]
Hello Paradajz.

Thanks for the great job you are doing with opendeck.

I'm beginning to understand your development, I've tried a raspberry pico board with several buttons, some fader, and an encoder.
Everything working perfectly.

But I have a question, is there a possibility to change the name of the box? I would like to be able to change the name to be able to differentiate the controllers, could you tell me where I can make this change?

**Comment by eurorackworld** [comment]
The name of the TARGET file your using during building the firmware is the name you will see.

So for example your config file is: Controller XYZ.yml 
Then the name of the midi device will be: OpenDeck - Controller XYZ

**Comment by paradajz** [comment]
You need to change the YAML filename, recompile the firmware and then flash it. The config filename is used as an USB name as well.

**Reply by itslemcy** [reply]
hi, on the webconfig i still have Custom OpenDeck board, but i would like to customise it, can you indicate me where to find this info?

**Reply by paradajz** [reply]
The board name is retrieved from here
https://github.com/shanteacontrols/OpenDeckUI/blob/master/src/definitions/board/boards.ts

The hardware UID retrieved via SysEx is then matched with that array.
https://github.com/shanteacontrols/OpenDeck/wiki/SysEx-Configuration#hardware-uid
